ansible/roles/kubernetes-addons/templates/kube-ui/dashboard-controller.yaml.j2

```diff
@@ -39,6 +39,6 @@ spec:
             port: 9090
           initialDelaySeconds: 30
           timeoutSeconds: 30
-    securityContext:
+      securityContext:
```

Comment:
When I `kubectl get` the securityContext is on the same level as `containers: []` or `volumes: []`.